javascript/attributes.js

```diff
@@ -83,3 +83,33 @@ export const findElement = attributes => {
   const element = elements.length === 1 ? elements[0] : null
   return element
 }
+
+const isTextInput = element => {
+  return (
+    [
+      'email',
+      'number',
+      'password',
+      'search',
+      'tel',
+      'text',
+      'textarea',
+      'url'
+    ].indexOf(element.type) >= 0
+  )
+}
```

Comment:
I might add an additional check for `element.tagName`

Reply:
Interesting. I thought they were basically the same. Hard to Google. Can you explain the difference in your own

Reply:
```javascript
document.getElementsByTagName("INPUT")[0].tagName
"INPUT"

document.getElementsByTagName("INPUT")[0].type
"hidden"
```

This feature should ignore all tags but `input` and `textarea`. Then only apply the logic to the types you've already outlined.